com.danielerolli.beavernotes.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.danielerolli.beavernotes
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: beaver-notes
+finish-args:
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+rename-desktop-file: beaver-notes.desktop
+rename-icon: beaver-notes
+modules:
+  - name: beaver-notes
+    buildsystem: simple
+    build-commands:
+      - ar -x Beaver-notes_3.0.0_amd64.deb
+      - tar -xf data.tar.xz
+      - mkdir -p $FLATPAK_DEST/opt
+      - cp -r opt/Beaver-notes/* $FLATPAK_DEST/opt
+      - mkdir -p $FLATPAK_DEST/share/applications
+      - cp -r usr/share/applications/* $FLATPAK_DEST/share/applications/
+      - mkdir -p $FLATPAK_DEST/share/icons/hicolor
+      - cp -r usr/share/icons/hicolor/* $FLATPAK_DEST/share/icons/hicolor/
+      - desktop-file-edit --set-key=Categories --set-value="X-Productivity;" $FLATPAK_DEST/share/applications/beaver-notes.desktop
+      - desktop-file-edit --set-key=Exec --set-value="beaver-notes %U" $FLATPAK_DEST/share/applications/beaver-notes.desktop
+      - install beavernotes.sh $FLATPAK_DEST/bin/beaver-notes
+    sources:
+      - type: file
+        url: https://github.com/Daniele-rolli/Beaver-Notes/releases/download/3.0.0/Beaver-notes_3.0.0_amd64.deb
```

Comment:
why not building from source?

Reply:
It's an Electron-based application, so it's hard to build from source. Other than that, the .deb file made it easier to understand where to place files and provided me with a .desktop file. Additionally, I've used Bitwarden's manifest as a sort of template, and since they did the same, I figured it wouldn't hurt anyone and would help me get it over with quickly.

Reply:
> I've used Bitwarden's manifest as a sort of template, 

two wrongs don't make a right.

Reply:
You might be right, but still, there's no way I can build it from source. I could swap the .deb with a tarball, but it would be the same.

Reply:
> there's no way I can build it from source.

Plenty of electron packages are built from source.

Reply:
I'll browse some repos and see what I can do. Since I have to rewrite the build process, I might as well add the ARM version.

Reply:
Look, I've browsed through your repos and found multiple Electron applications that use a .deb or .AppImage in their build process. I'm all for adding support for arm64, and I will, since you've pointed it out. However, it doesn't seem fair that I have to rebuild the manifest from the ground up to build the app from source, while tons of Electron applications just do what I did and are on Flathub. Thanks for understanding.